### PR TITLE
docs: add release document

### LINKED ISSUE #1497: Docs: Create document regarding release process
### Is your enhancement related to a problem? Please describe

No documentation on how to compile / make a release

### Describe the solution you'd like

Should have a short document outlining the release process / any changes we need to make externally as well (brew, chocolately, etc.)

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by benoitf**
### What does this PR do?
Document process to cut a release

### Screenshot/screencast of this PR

N/A

### What issues does this PR fix or reference?

fixes https://github.com/containers/podman-desktop/issues/1497

### How to test this PR?

Change-Id: Ic19eb6323d2dc95e66f3c2046bf1f36512847654

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+Release process for Podman Desktop
+---
+
+## Pre-Requisites
+
+- Make sure everything that needs to be in the release is there.
```
Comment (cdrage) [inline RELEASE.md]:
```suggestion
- Make sure everything that needs to be in the release is there
```

File: RELEASE.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+- Notify main contributors
+- Check PR for Release Notes is almost complete
```
Comment (cdrage) [inline RELEASE.md]:
```suggestion
- Check that the PR for Release Notes is almost completed by PM
```

File: RELEASE.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+
+## Cut a new release using the release workflow
+
+1. Go to https://github.com/containers/podman-desktop/actions/workflows/release.yaml
+1. Click on the top right drop-down menu `Run workflow`
+1. Enter the name of the release. Example: `0.12.0` (do not use the v prefix like v0.12.0)
+1. specify the branch to use for the release. It's main for all major releases. For a bugfix release
```
Comment (cdrage) [inline RELEASE.md]:
```suggestion
1. Specify the branch to use for the release. It's main for all major releases. For a bugfix release
```

File: RELEASE.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+1. Click on the `Run workflow` button
+1. edit the version the milestone going to https://github.com/containers/podman-desktop/milestones and select the date and then it `Close milestone`
```
Comment (cdrage) [inline RELEASE.md]:
```suggestion
1. Edit the version the milestone going to https://github.com/containers/podman-desktop/milestones and select the date and then it `Close milestone`
```

File: RELEASE.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+1. Create a new milestone for the upcoming release for example `0.13.0` and click on `Create milestone`
+1. Follow release process from https://github.com/containers/podman-desktop/actions/workflows/release.yaml
+1. Approve PR for the upcoming version. Title looks like `chore: 📢 Bump version to 0.13.0`
+1. Edit the release https://github.com/containers/podman-desktop/releases/edit/v0.12.0
+1. Select previous tag like v0.11.0 and click on `Generate release notes` and the click on `Update release`
+
+Follow-up:
+
+## package manager workflow
```
Comment (cdrage) [inline RELEASE.md]:
```suggestion
## Package manager workflow
```

File: RELEASE.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+
+Pre-requisites:
+- Ensure the release is OK (green workflow, artifacts are there https://github.com/containers/podman-desktop/releases)
+
+#### Brew
+
+We can publish to brew. It creates a PR to brew cask repository https://github.com/Homebrew/homebrew-cask/
+1. Go to https://github.com/containers/podman-desktop/actions/workflows/publish-to-brew.yaml
+1. Click on the top right drop-down `Run workflow`
+1. select the release version like `0.12.0` and click `Run workflow`
```
Comment (cdrage) [inline RELEASE.md]:
```suggestion
1. Select the release version like `0.12.0` and click `Run workflow`
```

File: RELEASE.md [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+
+#### Winget
+
+We can publish to Winget. It creates a PR to winget repository https://github.com/microsoft/winget-pkgs/
+1. Go to https://github.com/containers/podman-desktop/actions/workflows/publish-to-brew.yaml
+1. Click on the top right drop-down `Run workflow`
+1. select the release version like `0.12.0` and click `Run workflow`
```
Comment (cdrage) [inline RELEASE.md]:
```suggestion
1. Select the release version like `0.12.0` and click `Run workflow`
```

File: RELEASE.md [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+
+
+#### Chocolatey
+
+We can publish to Chocolatey. It creates a new submission at https://community.chocolatey.org/packages/podman-desktop/#versionhistory
+1. Go to https://github.com/containers/podman-desktop/actions/workflows/publish-to-winget.yaml
+1. Click on the top right drop-down `Run workflow`
+1. select the release version like `0.12.0` and click `Run workflow`
```
Comment (cdrage) [inline RELEASE.md]:
```suggestion
1. Select the release version like `0.12.0` and click `Run workflow`
```

File: RELEASE.md [hunk 9]
```diff
@@ -0,0 +1,96 @@
...
+1. Fork repository https://github.com/flathub/io.podman_desktop.PodmanDesktop and clone your repository (example https://github.com/benoitf/io.podman_desktop.PodmanDesktop)
+1. Edit the file io.podman_desktop.PodmanDesktop.yml
+1. Replace url: https://github.com/containers/podman-desktop/archive/refs/tags/vXXX.tar.gz by the correct version
+1. Update the SHA256 using for example `shasum -a 256 v0.12.0.tar.gz` command
+1. Unpack the v0.12.0.tar.gz (like `tar zxf v0.12.0.tar.gz`)
+1. Run the command (updating the volume mount path)
+   ```sh
+   podman run --rm -it -v $(pwd)/podman-desktop-0.12.0:/podman quay.io/podman-desktop/flatpak-node-generator yarn /podman/yarn.lock -o /podman/generated-sources.json
+   ```
+1. copy the file `$(pwd)/podman-desktop-0.12.0/generated-sources.json` to `generated-sources.json`
```
Comment (cdrage) [inline RELEASE.md]:
```suggestion
1. Copy the file `$(pwd)/podman-desktop-0.12.0/generated-sources.json` to `generated-sources.json`
```

File: RELEASE.md [hunk 10]
```diff
@@ -0,0 +1,96 @@
...
+1. commit the files `generated-sources.json` and `io.podman_desktop.PodmanDesktop.yml` and create a PR to the repository with a title like `feat: bump to v0.11.0`
```
Comment (cdrage) [inline RELEASE.md]:
```suggestion
1. Commit the files `generated-sources.json` and `io.podman_desktop.PodmanDesktop.yml` and create a PR to the repository with a title like `feat: bump to v0.11.0`
```

File: RELEASE.md [hunk 11]
```diff
@@ -0,0 +1,96 @@
...
+1. If PR is successful, merge the PR
```
Comment (cdrage) [inline RELEASE.md]:
```suggestion
1. If the PR is successful, merge the PR
```

File: RELEASE.md [hunk 12]
```diff
@@ -0,0 +1,96 @@
...
+### Announcement
+
+#### Release notes
+
+1. Check the PR in podman-desktop repository and merge it (title should be like `Release Notes 0.12`)
+
+#### Github discussions
+
+1. Create a new announcement at https://github.com/containers/podman-desktop/discussions
+1. title : Podman Desktop v0.12.0
```
Comment (cdrage) [inline RELEASE.md]:
```suggestion
1. Title : Podman Desktop v0.12.0
```

**Review by cdrage — CHANGES_REQUESTED** [review]
Honestly these changes are minor as this is more of an internal document / not really public facing. So I'm good with no grammatical changes / etc.

**Review by jeffmaury — APPROVED** [review]
LGTM

**Comment by benoitf** [comment]
suggestions applied

**Review by cdrage — APPROVED** [review]

**Pull request merged**